README.md

```diff
@@ -109,8 +109,10 @@ The `services` section lists the services that the host offers, such as
 providing an HTTP server. You must supply the `protocol`, `transport` and `port`
 values for each service. You may also specify `weight` and/or `host`.  They each
 default to a zero value. Please consult the RFC for an explanation of these
-values. Services can be configured in `config.exs` as shown above, or at
-runtime:
+values. You can also specify `payload` which is used to define the data in a TXT
```

Comment:
I'm not familiar with the proper naming of this, but I'm wondering if there will be other payloads in the future. Like maybe this should be `txt` or `txt_payload`. (If you don't know, I'll skim through the RFC to see if it has any naming guidance.)

Reply:
I also don't know what could be an appropriate name, `txt_payload` seems a really nice suggestion.

Reply:
👍 for `txt_payload`

Reply:
I don't see anything better in the RFC. Sounds like we're going with `txt_payload`. @eduardo-cunha-bose, could you update the PR?